rectools/metrics/intersection.py

```diff
@@ -0,0 +1,71 @@
+from typing import Optional
+
+import attr
+import pandas as pd
+
+from rectools import Columns
+from rectools.metrics.base import MetricAtK
+from rectools.metrics.classification import Recall
+
+
+@attr.s
+class Intersection(MetricAtK):
+    """
+    Metric to measure intersection in user-item (or item-item) pairs between recommendation lists.
```

Comment:
There is no support for item-item case in the code. So let's delete this from the docstring before it is implemented.